clang/test/Driver/sycl-offload-with-split.c

```diff
@@ -296,6 +296,16 @@
 // RUN:    | FileCheck %s -check-prefixes=CHK-NO-SPLIT
 // CHK-NO-SPLIT-NOT: sycl-post-link{{.*}} -split{{.*}}
 
+// Check no device code split mode is passed to sycl-post-link when -fsycl-device-code-split is not set and the target is FPGA
+// RUN:   %clang -### -fsycl -fsycl-targets=spir64_fpga-unknown-unknown-sycldevice %s 2>&1 | FileCheck %s -check-prefixes=CHK-NO-SPLIT
```

Comment:
Maybe skip sycldevice triple part if this is going to be dropped soon?

Reply:
It's already dropped.
https://github.com/intel/llvm/commit/29a1f167060e69d4d90d1e6a15d4998fd4556c4d

Reply:
Removed 'sycldevice' from triple.